### PR TITLE
Fixes for cross-browser issues

### PR DESCRIPTION
**Pull request by bishsbytes**
SASS / CSS Issues
---
- Some SASS dependencies were missing and references incorrect
- Inline SVG was not encoded, meaning the image wasn't showing in compliant browsers e.g. FireFox

JS Event Issue
---
- Labels not dynamic in IE11 and Edge

The input event is not supported for select elements in IE & Edge (https://developer.mozilla.org/en-US/docs/Web/Events/input#Browser_compatibility).

Whilst it isn't obvious this library isn't firing on select input when using styles 1 & 2, I've been using the default style so it was clear the label wasn't appearing after selecting an option when using IE11.

I've added a check for select elements + IE / Edge when attaching events.

### CODE REVIEW

File: src/float-labels.js [hunk 1]
```diff
@@ -169,9 +169,9 @@
 		handleEvents: function( el, action )
 		{
 			var events = this.events;
-			var isIE = this.isIe();
+			var isMS = this.isMSBrowser();
```
Comment (pryley) [inline src/float-labels.js]: Shouldn't this be something like:

```js
var isMicrosoftSelectEl = el.nodeName === 'SELECT' && this.isMSBrowser();
['blur','input','focus'].forEach( function( event ) {
    if( event === 'input' && (el.type === 'file' || isMicrosoftSelectEl )) {
        // ...
    }
});
```

**Comment by pryley** [comment]
Looks good. I'll review and merge later this evening.

**Comment by bishsbytes** [comment]
@pryley incorporated your suggestion.

Also noticed since fixing the SVG encoding that the native select arrows on IE needed to be hidden, so I've added this as well.

**Comment by bishsbytes** [comment]
👍

**Pull request merged**